.github/workflows/ci.yaml

```diff
@@ -0,0 +1,138 @@
+name: CI
```

Comment:
Actually now that it exists, let's just use the workflows from https://github.com/Workiva/gha-dart-oss